templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,13 @@
 {% extends "base.html" %}
 
-{% block content %}
-    <h1>Manufacturer List
-    </h1>
-
-    {% if manufacturer_list %}
-      <table class="table">
-        <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
-        </tr>
+{% load crispy_forms_filters %}
 
-        {% for manufacturer in manufacturer_list %}
-            <tr>
-              <td>
-                  {{ manufacturer.id }}
-              </td>
-              <td>
-                  {{ manufacturer.name }}
-              </td>
-              <td>
-                  {{ manufacturer.country }}
-              </td>
-          </tr>
-        {% endfor %}
-      </table>
+{% block content %}
+  <h1>{% if object %}Update{% else %}Create{% endif %} Manufacturer</h1>
```

Comment:
The logic for determining whether to display 'Update Manufacturer' or 'Create Manufacturer' is based on the presence of `object`. Ensure this logic accurately reflects the intended context, possibly by checking if the form is bound or by using a more explicit condition.